src/NzbDrone.Core/History/History.cs

```diff
@@ -41,6 +41,8 @@ public enum HistoryEventType
         DownloadFolderImported = 3,
         DownloadFailed = 4,
         TrackFileDeleted = 5,
-        TrackFileRenamed = 6
+        TrackFileRenamed = 6,
+        AlbumImportIncomplete = 7,
+        DownloadComplete = 8
```

Comment:
Whats the difference in this and downloadFolderImported?

Reply:
DownloadFolderImported should really be renamed to TrackFileImported i.e. it's logged for every track imported.  Previously one of these events was sufficient to mark a download as imported, but now there's a new event to mark that.